### PR TITLE
fix: (Core) form adjust hint alignment

### LINKED ISSUE #5155: FormGroup: Label / Hint formatting
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
I am not sure if this is by design label together with hint looks pretty strange . Please see pics (description field

Screen Shot 2021-04-18 at 1 18 42 PM

### PR DESCRIPTION
**Pull request by sKudum**
#### Please provide a link to the associated issue.
Closes #5155 

#### Please provide a brief summary of this pull request.
As hint alignment does not have Fiori specification. UX designers has suggested it has to be aligned as below.

Before:

After:

#### Please check whether the PR fulfills the following requirements
- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a ] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
     "karma-viewport": "1.0.8",
     "moment": "2.29.1",
     "ngx-markdown": "10.1.1",
-    "npm": "7.11.1",
+    "npm": "7.11.2",
```
Comment (KevinOkamoto) [inline package.json:122]: Do we need `npm` as a dependency?

Comment (sKudum) [reply package.json]: Removed it

**Review by droshev — CHANGES_REQUESTED** [review]
Can you remove package-lock.json from the PR

**Comment by InnaAtanasova** [comment]
I noticed a couple of styling problems:
1. In Core Textarea component the question mark icon is cut
Screen Shot 2021-05-10 at 11 38 34 AM
2. Icon and text are not vertically aligned
Screen Shot 2021-05-10 at 11 39 19 AM
3. When the text is very long, the icon is neever aligned with the right end of the textarea. There's some padding+margin spacing that prevents the icon reaching the end of the container.
Screen Shot 2021-05-10 at 11 40 37 AM

**Comment by sKudum** [comment]
> I noticed a couple of styling problems:
> 
> 1. In Core Textarea component the question mark icon is cut
> 
> Screen Shot 2021-05-10 at 11 38 34 AM
> 

==>Updated

> 2. Icon and text are not vertically aligned
> Screen Shot 2021-05-10 at 11 39 19 AM
> Updated


> 3. When the text is very long, the icon is neever aligned with the right end of the textarea. There's some padding+margin spacing that prevents the icon reaching the end of the container.
> Screen Shot 2021-05-10 at 11 40 37 AM

This has been added explicitly as part of https://github.com/SAP/fundamental-ngx/pull/4061/..I have made them 0rem for now. Let me know do I need have it updated or not.

**Comment by mikerodonnell89** [comment]
Looks good, I'm wondering if because the text has some margin-right, I'm wondering if the icon should have some margin-right when the icon comes before the text.  Also wondering if the icon should be the same color as the label color

Screen Shot 2021-05-12 at 7 42 30 AM

**Comment by sKudum** [comment]
> Looks good, I'm wondering if because the text has some margin-right, I'm wondering if the icon should have some margin-right when the icon comes before the text. Also wondering if the icon should be the same color as the label color
> 
> Screen Shot 2021-05-12 at 7 42 30 AM

Hi Mike, we don't have fiori specifications for it, so not sure what value i need to add exactly.

**Review by InnaAtanasova — APPROVED** [review]

**Comment by dimamarksman** [comment]
@sKudum preview is broken

**Comment by dimamarksman** [comment]
@sKudum e2e tests are failing, please take a look

**Review by Betrozov — APPROVED** [review]

**Comment by sKudum** [comment]
Hi @droshev , @InnaAtanasova , @Betrozov , QA team has pushed e2e changes please do have a these are helping to resolve test case build issues. Please do provide your comments on it.

**Review by droshev — APPROVED** [review]

**Pull request merged**